diff_test.go

```diff
@@ -236,3 +239,307 @@ func TestDiffBlobs(t *testing.T) {
 		t.Fatalf("Bad number of lines iterated")
 	}
 }
+
+func Test_ApplyDiff_Addfile(t *testing.T) {
```

Comment:
```suggestion
func TestApplyDiffAddfile(t *testing.T) {
```

so far, no tests have underscores in their names.